src/main/scala/dx/api/DxApi.scala

```diff
@@ -591,6 +597,21 @@ case class DxApi(logger: Logger = Logger.Quiet, dxEnv: DXEnvironment = DXEnviron
     content
   }
 
+  private def triageOne(components: DxPathComponents): Either[DxDataObject, DxPathComponents] = {
+    if (isDataObjectId(components.name)) {
+      val dxDataObj = dataObject(components.name)
+      val dxObjWithProj = components.projName match {
```

Comment:
Should the names be: dxDataObj and dxDataObjWithProject? 
Or dxObj and dxObjWithProj (so that it's clear the difference between them is only the project information on the latter one)

Reply:
renamed the second to dxDataObjWithProj